webserver-darwin.sh

```diff
@@ -1,36 +1,54 @@
-#!/usr/bin/zsh
+#!/bin/bash
 
-echo "brew should be installed"
+# Install homebrew
+  echo "brew should be installed"
 if ( which brew )
 then
   echo "brew already installed"
 else
   echo "installing brew"
   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
Is this non-interactive when it runs? Is there a way to make this non-interactive?

Reply:
yes i believe so, i will work on that.